static/src/javascripts/projects/commercial/modules/header-bidding/prebid/bid-config.js

```diff
@@ -289,6 +291,19 @@ const getTripleLiftInventoryCode = (
     return '';
 };
 
+const getOzoneTargeting = (): { } => {
+    const lotameData: LotameData = getLotameData();
+
+    if (typeof lotameData !== 'undefined') {
+        return {
+            ...buildAppNexusTargetingObject(getPageTargeting()),
```

Comment:
You may want to extract `buildAppNexusTargetingObject` into a `const` as it's return either way.